Darkly/darkly.xcdatamodeld/.xccurrentversion

```diff
@@ -1,5 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Since this is removed, should I also clean up https://github.com/launchdarkly/ios-client/blob/master/LaunchDarkly.podspec#L103-L107 ?